packages/cli/test/integration/auth.api.test.ts

```diff
@@ -17,17 +17,14 @@ let globalMemberRole: Role;
 let authAgent: AuthAgent;
 
 beforeAll(async () => {
-	app = await utils.initTestServer({ endpointGroups: ['auth'], applyAuth: true });
 	const initResult = await testDb.init();
 	testDbName = initResult.testDbName;
+	app = await utils.initTestServer({ endpointGroups: ['auth'], applyAuth: true });
```

Comment:
Since this had to be done in all tests, maybe we could check in `initTestServer` that `Db` is initialized to prevent out-of-order calls, unless the default error message for this is explicit enough.